plugins/woocommerce/tests/php/includes/class-wc-checkout-test.php

```diff
@@ -97,6 +97,34 @@ public function test_order_notes() {
 		$this->assertStringContainsString( 'This text should not save inside an anchor.', $content );
 	}
 
+	/**
+	 * @testdox the customer notes can have linebreaks.
+	 */
+	public function test_order_notes_linebreaks() {
+		$data = array(
+			'ship_to_different_address' => false,
+			'order_comments'            => 'A string
+			with linebreaks
+			in it.',
```

Comment:
Can we make this less indent weird and use the PHP constant to work across systems? (Never know if tests move to windows 😂 )
```suggestion
			'order_comments'            => 'A string' . PHP_EOL . 'with linebreaks' . PHP_EOL . 'in it.',
```

Reply:
Good idea thanks!